README.md

```diff
@@ -5,7 +5,7 @@
 ## Requirements
 
 In order to use `copilot-ops`, you need to have an OpenAI account with access to the GPT-3 Codex model,
-and an API token saved as the `OPENAI_API_TOKEN` environment variable.
+and an API token saved as the `OPENAI_API_TOKEN` environment variable. You must also have golang installed to build the binary.
```

Comment:
The beautiful thing about Golang is that it is *not* required to run Go binaries

Reply:
![image](https://user-images.githubusercontent.com/70029899/175133335-ee20d394-68f5-44f3-87a5-7984624966c4.png)